docs/guides/compute-daemons/readme.md

```diff
@@ -57,8 +57,9 @@ The command line arguements can be provided to the service definition or as an o
 | `--kubernetes-service-port=[PORT]` | The listening port of the kubeapi server |
 | `--service-token-file=[PATH]` | Location of the service token file |
 | `--service-cert-file=[PATH]` | Location of the service certificate file |
-| `--node-name=[COMPUTE-NODE-NAME]` | Name of this compute node as described in the System Configuration |
-| `--nnf-node-name=[RABBIT-NODE-NAME]` | Name of the rabbit node connected to this compute node as described in the System Configuration (`nnf-dm` daemon only)|
+| `--node-name=[COMPUTE-NODE-NAME]` | Name of this compute node as described in the System Configuration. Defaults to the os.Hostname() value. |
```

Comment:
Might want to be explicit and say that it defaults to the hostname of the compute node rather than `os.Hostname()`.